pasqal_cloud/utils/filters.py

```diff
@@ -76,6 +76,46 @@ def status_enum_to_string(self, job_statuses: List[JobStatus]) -> List[str]:
         return [job_status.value for job_status in job_statuses]
 
 
+class CancelJobFilters(BaseFilters):
+    """
+    Class to provide filters for cancelling a group of jobs.
+
+    Setting a value for any attribute of this class will add a filter to the query.
+
+    When using several filters at the same time, the API will return elements who pass
+    all filters at the same time:
+        - If the filter value is a single element, the API will return jobs whose
+          attribute matches the provided value.
+        - If the filter value is a list, the API will return jobs whose value for
+          this attribute is contained in that list.
+
+    Attributes:
+        id: Filter by job IDs
+        min_runs: Minimum number of runs.
+        max_runs: Maximum number of runs.
+        start_date: Retry jobs created at or after this datetime.
+        end_date: Retry jobs created at or before this datetime.
+    """
+
+    id: Optional[Union[List[Union[UUID, str]], Union[UUID, str]]] = None
+    min_runs: Optional[int] = None
+    max_runs: Optional[int] = None
+    start_date: Optional[datetime] = None
+    end_date: Optional[datetime] = None
```

Comment:
`RebatchFilters`, `CancelJobFilters`, and `SuspendJobFilters` share some filter attributes. Can we move these shared attributes to `BaseFilters`?

Reply:
Indeed it is possible, I will merge it